### PR TITLE
Feedback Form component

### PR DESCRIPTION
**Pull request by Lvlad27**
- Added Feedback Form component
- Webpack index.html source file change
- dataService file restructure

image

### CODE REVIEW

File: src/Components/FeedbackPage/FeedbackForm/FeedbackForm.jsx [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+        ...prevFormData,
+        [name]: type === 'checkbox' ? checked : value
+      };
+    });
+  }
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    setFormErrors(validate(formData));
+    setIsSubmit(true);
```
Comment (JuliaShavyrova) [inline src/Components/FeedbackPage/FeedbackForm/FeedbackForm.jsx]: why not just save to local storage when `validate(formData)` returns empty object? in this case you wouldn't need isSubmit state at all

File: src/Components/FeedbackPage/FeedbackForm/FeedbackForm.jsx [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+    if (!values.comments) {
+      errors.comments = '❌ Please leave a comment!';
+    } else if (!values.comments.length < 20) {
+      errors.comments = '❌ Must be at least 20 characters';
+    }
+    if (!values.feature) {
+      errors.feature = '❌ Please select a feature!';
+    }
+    if (values.feedbackType === 'selection' || !values.feedbackType) {
+      errors.feedbackType = '❌ Please select feedback type!';
```
Comment (JuliaShavyrova) [inline src/Components/FeedbackPage/FeedbackForm/FeedbackForm.jsx]: i would recommend to move the cross icon into ErrorMessage component as far as it's not dynamic and should be shown for any error

File: src/Components/FeedbackPage/FeedbackForm/FeedbackForm.jsx [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+          <input
+            type="radio"
+            id="forecast"
+            name="feature"
+            value="Forecast"
+            checked={formData.feature === 'Forecast'}
+            onChange={handleChange}
+          />
+          <label htmlFor="forecast">Forecast</label>
+          <br />
```
Comment (JuliaShavyrova) [inline src/Components/FeedbackPage/FeedbackForm/FeedbackForm.jsx]: br tag is usually used for text formatting (inside `<p></p>` or `<span></span>` tags) not blocks, so would be better to use css instead in this case

File: src/Components/FeedbackPage/FeedbackForm/FeedbackForm.jsx (hunk 1 above)
Comment (Lvlad27) [reply src/Components/FeedbackPage/FeedbackForm/FeedbackForm.jsx]: I agree. Removed isSubmit state.

File: src/Components/FeedbackPage/FeedbackForm/FeedbackForm.jsx (hunk 2 above)
Comment (Lvlad27) [reply src/Components/FeedbackPage/FeedbackForm/FeedbackForm.jsx]: done

File: src/Components/FeedbackPage/FeedbackForm/FeedbackForm.jsx (hunk 3 above)
Comment (Lvlad27) [reply src/Components/FeedbackPage/FeedbackForm/FeedbackForm.jsx]: done

**Review by JuliaShavyrova — APPROVED** [review]

**Pull request merged**